src/pages/patientView/clinicalInformation/lib/clinicalAttributesUtil.js

```diff
@@ -282,9 +325,8 @@ function getSpanElementsFromCleanData(
                 <span
                     is
                     class="clinical-attribute"
-                    attr-id={key}
```

Comment:
Removing attr-id and study along with  lines 21-24 of the CSS makes the patient view much more verbose

Reply:
Restoring the attr-id, study HTML attributes and CSS lines 21-24 makes the patient view closer to the reference. However the "Copy #" is still missing from under the "33 Mutations" header.

Reply:
I refreshed https://www.cbioportal.org/patient?studyId=ucec_tcga_pub&caseId=TCGA-BK-A0CC and it looks like the "Copy #" column is gone from there too, so please disregard my comments about it above!